GitUI/Translation/English.Plugins.xlf

```diff
@@ -563,10 +563,6 @@ Are you sure you want to continue?</source>
         <source>&amp;Publish</source>
         <target />
       </trans-unit>
-      <trans-unit id="PublishType.Text">
```

Comment:
Changes to the release branch should not include translations. Can this be reverted?